kodilla-testing/src/test/java/com/kodilla/testing/collection/CollectionTestSuite.java

```diff
@@ -0,0 +1,52 @@
+package com.kodilla.testing.collection;
+
+import com.kodilla.testing.collection.OddNumbersExterminator;
+import org.junit.*;
+import java.util.*;
+
+public class CollectionTestSuite {
+
+    @Before
+    public void before(){
+        System.out.println("Testing new case:");
+    }
+
+    @After
+    public void after(){
+        System.out.println("Test finished");
+    }
+
+    @Test
+    public void testOddNumbersExterminatorEmptyList(){
+        //Given
+        ArrayList<Integer> startList = new ArrayList<Integer>();
+        OddNumbersExterminator exterminator = new OddNumbersExterminator();
```

Comment:
`OddNumbersExterminator exterminator = new OddNumbersExterminator();`
to wystepuje w 2 testach. Mozesz to wyciaganac do zmiennych klasowych, aby nie powielac kodu